discordbot/commands/explain.py

```diff
@@ -95,7 +104,7 @@ async def explain(ctx: MtgContext, *, thing: Optional[str]) -> None:
             {}
         ),
         'prices': (
-            f"""
```

Comment:
I think if we lose the `f` here the vars in curly braces in this string will no longer be interpolated.

Reply:
Yep

Reply:
Got it.